src/status_im2/common/scroll_page/view.cljs

```diff
@@ -102,32 +104,41 @@
           :style  style/display-picture}]]))])
 
 (defn scroll-page
-  [cover page-nav name]
+  [{:keys [background-color cover page-nav content-container name]}]
   (let [scroll-height (reagent/atom negative-scroll-position-0)]
     (fn [sticky-header children]
       [:<>
-       [scroll-page-header scroll-height name page-nav cover sticky-header]
+       [scroll-page-header
+        {:scroll-height scroll-height
+         :name          name
+         :page-nav      page-nav
+         :cover         cover
+         :sticky-header sticky-header}]
        [rn/scroll-view
         {:style                           (style/scroll-view-container
                                            (diff-with-max-min @scroll-height 16 0))
          :shows-vertical-scroll-indicator false
-         :scroll-event-throttle           16
+         :scroll-event-throttle           8
```

Comment:
Curious, did you notice a more smooth animation when moving to 8ms? The ReactNative documentation recommends against reducing this number below 16, so unless you really saw a noticeable difference it'd be better to keep it at 16 or increase to 32, 64, etc.

>You will not notice a difference between values set between 1-16 as the JS run loop is synced to the screen refresh rate. If you do not need precise scroll position tracking, set this value higher to limit the information being sent across the bridge
>
>-- https://reactnative.dev/docs/scrollview#scrolleventthrottle-ios

Reply:
not really there wasnt any noticeable change. I will revert this to 16. Thanks @ilmotta